sdk/src/test/scala/io/horizen/account/utils/AccountMockDataHelper.scala

```diff
@@ -437,10 +437,10 @@ case class AccountMockDataHelper(genesis: Boolean)
   private def setupMockMessageProcessor = {
     val mockMsgProcessor = mock[MessageProcessor]
     Mockito
-      .when(mockMsgProcessor.canProcess(any[Message], any[BaseAccountStateView]))
+      .when(mockMsgProcessor.canProcess(any[Message], any[AccountStateView], any[Int]))
```

Comment:
This should be a BaseAccountStateView, not an AccountStateView, same below.

Reply:
Done